t/homology.t

```diff
@@ -59,57 +59,51 @@ my $condensed_ortho_ENSG00000139618_gorilla = {
 );
 
 is_json_GET(
-    '/homology/id/ENSG00000139618?format=condensed;target_species=gorilla_gorilla;type=orthologues', {data => []},
+    '/homology/id/homo_sapiens/ENSG00000139618?format=condensed;target_species=gorilla_gorilla;type=orthologues', {data => []},
     'homologies without compara division specified',
 );
 
 is_json_GET(
-    '/homology/id/ENSG00000139618?compara=homology;format=condensed;target_taxon=9595;type=orthologues',
+    '/homology/id/homo_sapiens/ENSG00000139618?compara=homology;format=condensed;target_taxon=9595;type=orthologues',
     _get_returned_json('ENSG00000139618', $condensed_ortho_ENSG00000139618_gorilla),
     '"condensed" homologies with a target single-species taxon',
 );
 
 is_json_GET(
-    '/homology/id/ENSG00000139618?compara=homology;format=condensed;target_taxon=9595;target_species=gorilla_gorilla;type=orthologues',
+    '/homology/id/homo_sapiens/ENSG00000139618?compara=homology;format=condensed;target_taxon=9595;target_species=gorilla_gorilla;type=orthologues',
     _get_returned_json('ENSG00000139618', $condensed_ortho_ENSG00000139618_gorilla),
     '"condensed" homologies with overlapping targets',
 );
 
 $json = json_GET(
-    '/homology/id/ENSG00000139618?compara=homology;format=condensed;target_species=pan_troglodytes;target_species=gorilla_gorilla;type=orthologues',
+    '/homology/id/homo_sapiens/ENSG00000139618?compara=homology;format=condensed;target_species=pan_troglodytes;target_species=gorilla_gorilla;type=orthologues',
     '"condensed" homologies with non-overlapping targets',
 );
 is(scalar(@{$json->{data}->[0]->{homologies}}), 2, 'The "target_species" and "target_taxon" arguments are combined');
 
 $json = json_GET(
-    '/homology/id/ENSG00000139618?compara=homology;format=condensed;target_taxon=207598;type=orthologues',
+    '/homology/id/homo_sapiens/ENSG00000139618?compara=homology;format=condensed;target_taxon=207598;type=orthologues',
     '"condensed" homologies with an ancestral taxon',
 );
 is(scalar(@{$json->{data}->[0]->{homologies}}), 2, 'Found both species under taxon ID 207598');
 
 $json = json_GET(
-    '/homology/id/ENSG00000139618?compara=homology;format=condensed;target_species=ornithorhynchus_anatinus;type=orthologues',
+    '/homology/id/homo_sapiens/ENSG00000139618?compara=homology;format=condensed;target_species=ornithorhynchus_anatinus;type=orthologues',
     '"condensed" homologies with a single target species but one-to-many homologies',
 );
 is(scalar(@{$json->{data}->[0]->{homologies}}), 2, 'We get all the one-to-many homologies even though there is 1 species');
 
 
 ## Total number of homologies without a filter
 $json = json_GET(
-    '/homology/id/ENSG00000139618?compara=homology;format=condensed;type=orthologues',
+    '/homology/id/homo_sapiens/ENSG00000139618?compara=homology;format=condensed;type=orthologues',
     '"condensed" homologies with no species filter',
 );
 is(scalar(@{$json->{data}->[0]->{homologies}}), 63, 'Got all the homologies');
```

Comment:
I think this unit test may be the same as the one immediately following it. If the two unit tests are identical, would it be possible to delete one of them?